base/acme/conf/validators.conf

```diff
@@ -0,0 +1,2 @@
+dns-01.class=org.dogtagpki.acme.validator.DNS01Validator
+http-01.class=org.dogtagpki.acme.validator.HTTP01Validator
```

Comment:
Should we simplify this further? For example:
```
dns-01=org.dogtagpki.acme.validator.DNS01Validator
http-01=org.dogtagpki.acme.validator.HTTP01Validator
```
and if we need params we can do:
```
dns-01=org.dogtagpki.acme.validator.DNS01Validator
dns-01.param1=value1
```

Reply:
I like `.class` because it gives you some expectation of what value you're wanting. You know you want a Java class that implements `dns-01`. Same holds for the other usages of `class` above. Keeps it consistent.